src/models/Group.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require("mongoose");
+
+const groupSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: [true, "please provide a name!"],
+    },
+    host: {
+      type: String,
+      required: [true, "a group must have a host!"],
+      unique: true,
```

Comment:
Could this be a `user` ref, rather than string?

Reply:
Yeah good idea, I'll modify it.